src/Plugin/StrawberryRunnersPostProcessorPluginBase.php

```diff
@@ -194,6 +194,43 @@ protected function kill($pid) {
     return strstr(PHP_OS, 'WIN') ? exec("taskkill /F /T /PID $pid") : exec("kill -9 $pid");
   }
 
+  /**
+   * Replace the last occurrence of a given value in the string.
+   * @see https://github.com/phannaly/laravel-helpers/blob/v1.0.3/src/String.php#L331
+   *
+   * @param  string  $search
+   * @param  string  $replace
+   * @param  string  $subject
+   * @return string
+   */
+  public function str_replace_last(string $search, string $replace, string $subject) {
```

Comment:
Do we use this at all?

Reply:
No, we don't but it is the matching function to the one we use. I see uses for that one soon

Reply:
I kinda liked the pattern from that library, maybe we could have a regular archipelago utils PHP library in the future with commonalities.